src/ontodia/data/sparql/responseHandler.ts

```diff
@@ -154,31 +154,25 @@ export function getElementsInfo(response: SparqlResponse<ElementBinding>, ids: s
     for (const sInst of sInstances) {
         let sInstTypeId: string = sInst.inst.value;
 
-        if (instancesMap[sInstTypeId]) {
-            enrichElement(instancesMap[sInst.inst.value], sInst);
-        } else {
-            instancesMap[sInstTypeId] = getElementInfo(sInst);
+        if (isThereAnyData(sInst)) {
+            if (instancesMap[sInstTypeId]) {
+                enrichElement(instancesMap[sInst.inst.value], sInst);
+            } else {
+                instancesMap[sInstTypeId] = getElementInfo(sInst);
+            }
         }
-    };
+    }
 
-    const proccesedIds = Object.keys(instancesMap);
-    for (const id of ids) {
-        if (proccesedIds.indexOf(id) === -1) {
-            instancesMap[id] = {
-                id: id,
-                label: { values: [getLocalizedString(undefined, id)] },
-                types: [THING_URI],
```

Comment:
`THING_URI` constant should be unused after this change, so it's better to remove it.